File: app/assets/javascripts/backbone/models/flight.js
var liger = liger || {};

liger.Flight = Backbone.Model.extend({
  urlRoot: function() {
    return '/airplanes/' + this.get(id) + '/flights'
  },
  defaults: {
    flight_no: null,
    departure: '',
    destination: '',
    date: '',
  }
});

File: app/models/flight.rb
# == Schema Information
#
# Table name: flights
#
#  id          :integer          not null, primary key
#  flight_no   :integer
#  departure   :string
#  destination :string
#  date        :string
#  plane_id    :integer
#  created_at  :datetime         not null
#  updated_at  :datetime         not null
#

class Flight < ActiveRecord::Base
  belongs_to :airplane
  has_many :reservations
end
